libs/components/core/src/lib/modules/viewkeeper/viewkeeper.ts

```diff
@@ -67,15 +67,17 @@ function setElPosition(
   left: number | string,
   top: number | string,
   width: number | string,
-  marginTop: number | string
+  marginTop: number | string,
+  clipTop: number | string,
+  clipLeft: number | string
 ): void {
   el.style.top = px(top);
   el.style.left = px(left);
   el.style.marginTop = px(marginTop);
+  el.style.clipPath =
+    clipTop || clipLeft ? `inset(${px(clipTop)} 0 0 ${px(clipLeft)})` : 'none';
 
-  /*istanbul ignore else*/
-  /* sanity check */
-  if (width !== null) {
+  if (width) {
```

Comment:
Before, this would allow a zero-width value. We don't want that anymore?

Reply:
Reverted this change.